packages/open-vm-tools/tools.conf

```diff
@@ -0,0 +1,5 @@
+[powerops]
+poweron-script=/usr/bin/true
```

Comment:
I'm assuming if we set these to `/usr/bin/true` the corresponding actions in the webUI for shutdown, poweroff, etc. won't do anything.

I think for resume/suspend it makes sense not to do anything, but what about `poweron` and `poweroff`?

Reply:
I verified that the actions work.

The default scripts are to run through a series of user or admin-defined commands before carrying out the power event. Since we don't support defining new ones, running `true` causes the event to be carried out immediately.

Reply:
Ah, gotcha.  I misunderstood that.